adafruit_bitmapsaver.py

```diff
@@ -91,6 +91,7 @@ def _write_pixels(output_file, pixel_source, palette):
                     color >>= 8
                     buffer_index += 1
         else:
+            result_buffer = bytearray(2048)
```

Comment:
How about using a for loop to set everything back to zero? As-is this will try and allocate a second (or more) 2k chunk. Setting the values will reuse the memory.

Reply:
Thanks for the nudge Scott to make this cleaner.  I wasn't sure which was the right number to iterate over, since there are a lot of different datatypes flying around in this one.

I think I have it right, by reinitializing `2 * width` elements in the `result_buffer`.  So that's two bytes per pixel as I belatedly noticed by the naming of `pixel565`.

Reply:
I think it is correct for resetting but I don't think you need to do this line anymore.

Reply:
My apologies for wasting your time, I hand-copied from a local copy and missed deleting this line. Many thanks for catching this.  

Note to self: Make a "submit PR" checklist item to read through the files "diff" before clicking submit on a PR.